### PR TITLE
use unix timestamps on tsdb request

### PR DESCRIPTION
**Pull request by vitorboschi**
this avoids issues when OpenTSDB is not running on UTC timezone. 
Fixes issue #806

### CODE REVIEW

**Comment by kylebrandt** [comment]
@mjibson See any issue with querying this way? I don't....

**Comment by mjibson** [comment]
Bosun does a lot dealing with time, so I'd like to test this a bit more before it goes in.

**Comment by mjibson** [comment]
This diff breaks our tests (`go test ./...` in the root directory) so it can't go in yet. It could be because the tests need to be updated for this new format

**Comment by mjibson** [comment]
Add https://github.com/bosun-monitor/bosun/commit/84c14cf0d0f0aaac3d640c55c441870b254288ec and we'll merge.

**Comment by vitorboschi** [comment]
@mjibson I've just updated the PR. Thanks for pointing and fixing the test

File: cmd/bosun/sched/sched_test.go [hunk 1]
```diff
@@ -112,7 +112,7 @@ func testSched(t *testing.T, st *schedTest) (s *Schedule) {
 }
 
 var queryTime = time.Date(2000, 1, 1, 12, 0, 0, 0, time.UTC)
-var window5Min = `"2000/01/01-11:55:00", "2000/01/01-12:00:00"`
+var window5Min = `"9.467277e+08", "9.46728e+08"`
```
Comment (captncraig) [inline cmd/bosun/sched/sched_test.go]: This makes the tests pretty hard to follow. Perhaps we can add a helper function to format dates properly, but still view them more sensibly?

Comment (mjibson) [inline cmd/bosun/sched/sched_test.go]: True. However, the tests were already doing this, except for the first query. At least they are consistently bad. We can make this improvement in another PR.
